__tests__/app.test.js

```diff
@@ -129,4 +130,36 @@ describe('4 get/api/articles/:article_id', () => {
         })
     });
   });
-  
\ No newline at end of file
+  
+  describe('6 post/api/articles/id/comments', () => {
+    test('returns 201, with the added comment', () => {
+        const comment = {
+            author: "lurker",
+            body : "Power of Water, Powers Unite! Six working together to fight evil!"
+        }
+        return request(app).post("/api/articles/4/comments").expect(201).send(comment).then((res)=>{
+          
+            expect(res.body.comment).toEqual(expect.objectContaining({
+                article_id: 4,
+                author: "lurker",
+                body : "Power of Water, Powers Unite! Six working together to fight evil!",
+                comment_id: expect.any(Number)
+            }))
+        })
+    })
+    test('returns 400, if article id is invalid', () => {
+        const comment = {
+            author: "lurker",
+            body : "Power of Water, Powers Unite! Six working together to fight evil!"
+        }
+        return request(app).post("/api/articles/stonks/comments").expect(400).send(comment).then((res)=>{
+            expect(res.body.msg).toBe("Bad, bad Request")
+        })
+    })
+    test('returns 400, if client tries to insert an empty object', () => {
+        const comment = {}
+        return request(app).post("/api/articles/stonks/comments").expect(400).send(comment).then((res)=>{
+            expect(res.body.msg).toBe("Bad, bad Request")
+        })
+    })
```

Comment:
Some more tests would be handy here, Arturo!

It's a prism of endless possibility, where a single choice can branch out into infinite realities, creating alternate worlds from the ones you know. I am the Watcher. I am your guide through these vast new realities. Follow me and ponder the question... **"What if?"**

- What if a post request is made to an article that doesn't exist? eg. `/api/articles/9999/comments`
- What if a post request is made by a user that doesn't exist in the db? eg. `{author: "Arturo", body: "Wow, have I been able to post? I'm not even registered on this site!"}`
- What if the article is valid but the comment object is empty or has missing keys? eg `{body: "I have no identity!"}`